services/QuillLMS/spec/models/concept_feedback_spec.rb

```diff
@@ -84,4 +74,25 @@
     end
   end
 
+  describe '#translations_json(language:)' do
+    subject { concept_feedback.translations_json(locale: )}
+
+    let(:locale) { Translatable::DEFAULT_LOCALE}
+
+    context 'there is a translation for the language' do
+      let(:concept_feedback) { create(:concept_feedback, :with_translated_text)}
```

Comment:
Again, I'd probably have a space at the end here so that you have a space inside both ends of the curly brace.  Same in the next context.

Reply:
I will make a rubocop for this and put it in another PR